Add governance doc (#927)
kubean-io/kubean: GOVERNANCE.md

# Kubean Project Governance

### User

Kubean community users, feel free to comment your usage information in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.

> The community welcomes any individuals, companies, and organizations to use, develop, feedback bugs, submit feature requests, and help each other. We recommend using [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue") to track bugs and manage requests.

Reviewer: ```suggestion
The community welcomes any individuals, companies, and organizations to use, contribute to, report bugs, submit feature requests, and help each other.
We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") to track bugs and manage requests.
```

### Contributor

Every user is welcome to participate and contribute to the Kubean open-source community in a number of ways, including but not limited to the following:

> 1. participate in the community by actively engaging in discussions in [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue").

Reviewer: ```suggestion
1. engage in lively discussions in [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues").
```

Author: 😁Thanks, @windsonsea, I'm done.

2. submit code patches.
3. translate, revise, add to, and improve the [documentation](https://kubean-io.github.io/kubean/ "Documentation");
4. share experiences with the Kubean project and actively promote it.
5. submit suggestions/criticisms.

Prominent contributors can expect to be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).
All contributors who have submitted PRs to Kubean will be permanently recorded in the [Contributors List](https://github.com/kubean-io/kubean#thumbsup-contributors).

### Core Technical Committee

> The Kubean project is governed and directed by a Core Technical Committee, which is responsible for the development of the Kubean project.

Reviewer: ```suggestion
The Kubean project is governed and directed by a Core Technical Committee (CTC), which is responsible for the development of the Kubean project.
```

> The CTC has final authority over this project including:

Reviewer: ```suggestion
The CTC has final authority over this project, which includes:
```

> * Technical direction
> * Project governance and process (including this policy)
> * Contribution policy
> * GitHub repository hosting
> * Conduct guidelines
> * Maintaining the list of additional Collaborators

Reviewer: ```suggestion
* Setting the technical direction
* Establishing project governance and processes (including this policy)
* Managing the contribution policy
* Overseeing the GitHub repository hosting
* Enforcing conduct guidelines
* Maintaining the list of additional Collaborators
```

The Core Technical Committee consists of Reviewers & Approvers, see **[OWNERS](https://github.com/kubean-io/kubean/blob/main/OWNERS "OWNERS")** for a list.

### Communication

> 1. we recommend [Github Issue](https://github.com/kubean-io/kubean/issues "Github Issue") for tracking bugs and managing requirements.
> 2. we recommend [Github Milestone](https://github.com/kubean-io/kubean/issues "GitHub Milestone") for managing project progress and planning;

Reviewer: ```suggestion
1. We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") for tracking bugs and managing requirements;
2. We recommend using [Github Milestone](https://github.com/kubean-io/kubean/issues "GitHub Milestone") for managing project progress and planning.
```

### Documentation
1. We recommend using [Github Pages](https://kubean-io.github.io/kubean "Github Pages") to deposit the documentation;

> 2. we recommend using [Github Wiki](https://github.com/kubean-io/kubean/wiki "Github Wiki") to deposit FAQs; `TODO`.

Reviewer: ```suggestion
2. We recommend using [Github Wiki](https://github.com/kubean-io/kubean/wiki "Github Wiki") to deposit FAQs; `TODO`.
```

# Kubean Project Governance

### User

> Kubean community users, feel free to comment your usage information in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.

Reviewer 2: ```suggestion
Kubean community users, feel free to feedback your use of Kubean in **[Who is using Kubean](https://github.com/kubean-io/kubean/issues/910 "Who is Using Kubean")**.
```

The community welcomes any individuals, companies, and organizations to use, contribute to, report bugs, submit feature requests, and help each other.

We recommend using [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues") to track bugs and manage requests.

### Contributor

Every user is welcome to participate and contribute to the Kubean open-source community in a number of ways, including but not limited to the following:

1. engage in lively discussions in [Github Issues](https://github.com/kubean-io/kubean/issues "Github Issues").
2. submit code patches.
3. translate, revise, add to, and improve the [documentation](https://kubean-io.github.io/kubean/ "Documentation");

> 4. share experiences with the Kubean project and actively promote it.

Reviewer 2: ```suggestion
4. Share experiences with the Kubean project and actively promote it.
```

> 5. submit suggestions/criticisms.

Reviewer 2: ```suggestion
5. Submit suggestions/criticisms.
```

> Prominent contributors can expect to be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).

Reviewer 2: ```suggestion
Prominent contributors can be included in the Kubean project's [Maintainers List](https://github.com/kubean-io/kubean/blob/main/MAINTAINERS.md).
```

Author: hi @Michelle951, I'm done.😀

Reviewer 2: /lgtm
